modules/ROOT/pages/_partials/include-manage-proxies.adoc

```diff
@@ -34,10 +42,14 @@ sudo /opt/anypoint/runtimefabric/rtfctl get monitoring-proxy
 The output should match the value expected.
 // end::monitoring-proxy[]
 // tag::http-proxy[]
+. After setting up the proxy, validate the setup by running the `rtfctl validate` command. See xref:install-rtfctl.adoc#validate[rtfctl Validate] documentation.
 
 == Configure an HTTP Proxy for Services in Runtime Fabric
 
-You can configure HTTP and HTTPS proxies so that outbound components can communicate with the control plane. 
+You can configure HTTP and HTTPS proxies so that outbound components can communicate with the control plane.
```

Comment:
@hegderohit89 We have not tested the **https** proxy yet, shall we not mention it here.